### PR TITLE
Update account templates & validator app

### PR DESCRIPTION
**Pull request by Herrie82**
Signed-off-by: Herman van Hazendonk <github.com@herrie.org>

### CODE REVIEW

File: accounts/org.webosports.messaging.gadu/org.webosports.messaging.gadu.json [hunk 1]
```diff
@@ -1,16 +1,15 @@
 	{
 		"templateId": "org.webosports.messaging.gadu",
 		"loc_name": "Gadu Gadu",
-		"loc_subheader": "Account Settings",
```
Comment (Garfonso) [inline accounts/org.webosports.messaging.gadu/org.webosports.messaging.gadu.json]: Why did you delete this line? Probably a mistake?

Comment (Herrie82) [inline accounts/org.webosports.messaging.gadu/org.webosports.messaging.gadu.json]: You're right, it was not consistent. I think in the end we might need to drop it altogether, but undecided for now. Just want to get some initial work pushed for now so we can take it forward.
